azure-automation-plus-dsc-lab/nested/03.15.00.createDomainController.json

```diff
@@ -0,0 +1,145 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adsPrefix": {
+      "type": "string"
+    },
+    "azureRegion": {
+      "type": "string"
+    },
+    "adminUserName": {
+      "type": "string"
+    },
+    "adminPassword": {
+      "type": "securestring"
+    },
+    "adsAvSetId": {
+      "type": "string"
+    },
+    "ads03nicId": {
+      "type": "string"
+    },
+    "saName": {
+      "type": "string"
+    },
+    "domainName": {
+      "type": "string"
+    },
+    "dscArtifactsUrl": {
+      "type": "string"
+    },
+    "dscUrlSasToken": {
+      "type": "string"
+    }
+  },
+  "variables": {
+    "ads03name": "[concat(parameters('adsPrefix'),3)]",
+    "adsVmSize": "Standard_A1_v2",
+    "imagePublisher": "MicrosoftWindowsServer",
+    "imageOffer": "WindowsServer",
+    "imageSku": "2016-Datacenter-Server-Core-smalldisk",
+    "diskNameOs": "[toLower(concat(variables('ads03name'),'-syst'))]",
+    "saType": "Standard_LRS",
+    "diskNameData": "[toLower(concat(variables('ads03name'),'-data'))]",
+    "saUrlSuffix": ".blob.core.windows.net",
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#referencing-resource-properties) for how to handle storage account uris